__tests__/app.test.js

```diff
@@ -307,3 +307,70 @@ describe("GET /api/users", () => {
       });
   });
 });
+describe("GET /api/articles queries", () => {
+  test("200: returns array of articles objects with correct props sorted desc article_id", () => {
```

Comment:
ideally you should build these up first - just one with the topic specified, then one with the order specified and then one with the sort specified.